src/Extension/MemberExtension.php

```diff
@@ -17,6 +22,18 @@ class MemberExtension extends DataExtension
     ];
 
     private static $db = [
-        'DefaultRegisteredMethod' => 'Varchar',
+        'DefaultRegisteredMethodID' => 'Int',
     ];
+
+    public function getDefaultRegisteredMethod()
+    {
+        $id = $this->owner->DefaultRegisteredMethodID;
+
+        if (!$id) {
+            return null;
+        }
+
+        $methods = $this->owner->RegisteredMFAMethods();
+        return $methods->byID($id);
```

Comment:
Assuming this is a HasManyList, wouldn't `HasManyList::byID(null)` return null too, meaning you could skip the falsy check above?